### PR TITLE
PIM-5275: Translate unit of metrics

### PR DESCRIPTION
**Pull request by skeleton**
| Q                 | A
| ----------------- | ---
| Specs             | N
| Behats            | :white_check_mark: 
| Blue CI           | ?
| Changelog updated | N
| Review and 2 GTM  | :white_check_mark: 
| Micro Demo (PO)   | TODO
| Migration script  | N
| Tech Doc          | N

### CODE REVIEW

**Comment by jjanvier** [comment]
gtm

File: features/channel/edit_channel.feature [hunk 1]
```diff
@@ -47,3 +50,9 @@ Feature: Edit a channel
     And I am on the locales page
     Then the grid should contain 1 element
     And I should see locale "en_US"
+
+  Scenario: Successfully display the unit of metrics
+    Given I am logged in as "Julien"
+    And  I am on the "tablet" channel page
+    And I fill in the following information:
```
Comment (grena) [inline features/channel/edit_channel.feature]: Please add a small comment to explain, seen irl :D

**Comment by grena** [comment]
Small comment then GTM !

**Comment by solivier** [comment]
gtm

**Pull request closed without merging**